src/components/Header/Header.test.tsx

```diff
@@ -7,10 +7,11 @@ import * as React from 'react';
 import Header from './Header'
 import HeaderIdentity from './HeaderIdentity'
 import HeaderNavigation from './HeaderNavigation'
+import { isMainThread } from 'worker_threads';
```

Comment:
What is this import for? I don't see it being used.

Reply:
I have no idea where that came from. Removing. Thanks.

Reply:
Actually, I do know. {{vscode}} incorrectly auto-completed something and then auto-imported it. Thanks for catching.